v3/src/components/case-table/case-table.tsx

```diff
@@ -85,11 +87,11 @@ export const CaseTable = observer(function CaseTable({ setNodeRef }: IProps) {
         <div ref={setTableRef} className="case-table" data-testid="case-table">
           <div className="case-table-content" onScroll={handleHorizontalScroll}>
             {collections.map((collection, i) => {
-              const key = collection?.id || kChildMostTableCollectionId
+              const key = collection.id
               const parent = i > 0 ? collections[i - 1] : undefined
               return (
-                <ParentCollectionContext.Provider key={key} value={parent}>
-                  <CollectionContext.Provider key={key} value={collection}>
+                <ParentCollectionContext.Provider key={key} value={parent?.id}>
+                  <CollectionContext.Provider key={key} value={collection.id}>
```

Comment:
Is it right that both contexts have the same key?

Reply:
It seems that the outer `Provider` needs a key but the inner one does not, so we can just eliminate the second and then we don't have to answer that question. 😉